other/^SciQA.md

```diff
@@ -0,0 +1,7 @@
+# SciQA
+
+**SciQA** [[1]](#myfootnote1)</sup> is scientific QA benchmark for scholarly knowledge. leveraging ORKG.  The dataset os created by manually developed a set of 100 complex questions that are answerable with ORKG. Based on those questions, eight question templates are adopted to automatically generated further 2465 questions. In total the dataset is composed of 2565 questions, along with the SPARQL queries, answers, etc. The dataset is available on [huggingface](:https://huggingface.co/datasets/orkg/SciQA) and [zenodo](https://zenodo.org/doi/10.5281/zenodo.5845197). 
```

Comment:
Fixed Grammar:
**SciQA** [[1]](#myfootnote1)</sup> is a scientific QA benchmark for scholarly knowledge leveraging the Open Research Knowledge Graph (ORKG). The authors first manually developed a set of 100 complex questions that are answerable with ORKG. Based on those questions, eight question templates are adopted to automatically generate 2465 further questions. In total the dataset is composed of 2565 questions, along with the SPARQL queries, answers, etc. The dataset is available on [huggingface](:https://huggingface.co/datasets/orkg/SciQA) and [zenodo](https://zenodo.org/doi/10.5281/zenodo.5845197).